### DISCUSSION TITLE
how to detected in controller if the websockets is not running?

### DISCUSSION THREAD

**Discussion by constantinosergiou** [beyondcode/laravel-websockets#958] [Q&A]
e.g 
```
event(new NewMessageNotification($data)); 
```
make this error if the websockets not run

Pusher error: cURL error 7: Failed to connect to 127.0.0.1 port 6001: Connection refused (see https://curl.haxx.se/libcurl/c/libcurl-errors.html)

**Comment by sananAhmad** [comment]
php artisan websockets:server check this is working or not using call method
if not like this solution then artisan call websockets:server and assign it to variable then check this

**Comment by sohailahmad07** [comment]
i have the same issue my websocket server is running and when i try to send an event from the websocket dashboard
i get an error
```
message	"Pusher error: cURL error 7: Failed to connect to 127.0.0.1 port 6001 after 0 ms: Connection refused (see https://curl.haxx.se/libcurl/c/libcurl-errors.html)"
```

**Reply by mathbc** [reply]
I have a same problem

**Comment by zevlyn** [comment]
I searched but couldn't help much, so this is my current solution

```php
$pusher = Broadcast::pusher(config('broadcasting.connections.pusher'));
$pusher->getChannels();
return true;
```
